### DISCUSSION TITLE
Update user email without sending Supabase email template

### DISCUSSION THREAD

**Discussion by riccardolardi** [supabase/supabase#29923] [Questions] [answered]
Hi, we are not using the built-in email template system from Supabase but an own email sending system. We would like to let users change their email address by using `supabase.auth.updateUser({email: "new@new.com"})` but then compose the verification link and send the email by ourselves. How can we disable Supabase to send the verify email?

**Comment by GaryAustin1** [comment, marked as answer]
You probably want to use this method to change the email then to get the info to send your own email.  https://supabase.com/docs/reference/javascript/auth-admin-generatelink

**Reply by riccardolardi** [reply]
Thanks! I missed that 👍
